accepted/PSR-7-http-message-meta.md

```diff
@@ -252,26 +252,24 @@ request or reset request state for each message the client sends:
 
 ~~~php
 $uri = new Uri('http://api.example.com');
-$baseRequest = new Request($uri, null, [
-    'Authorization' => 'Bearer ' . $token,
-    'Accept'        => 'application/json',
-]);;
+$baseRequest = new Request(
+    $uri, 
+    null, 
+    ['Authorization' => 'Bearer ' . $token, 'Accept' => 'application/json']
+);
 
 $request = $baseRequest->withUri($uri->withPath('/user'))->withMethod('GET');
 $response = $client->send($request);
 
 // get user id from $response
 
-$body = new StringStream(json_encode(['tasks' => [
-    'Code',
-    'Coffee',
-]]));;
+$body = new StringStream(json_encode(['tasks' => ['Code', 'Coffee']]));
```

Comment:
```suggestion
$body = new StringStream(
    json_encode(
        [
            'tasks' => [
                'Code', 
                'Coffee',
            ]
        ]
    )
);
```

Reply:
> Lines SHOULD NOT be longer than 80 characters; lines longer than that SHOULD be split into multiple subsequent lines of no more than 80 characters each.

whats wrong in current example?

Reply:
It's just to be consisted with the rest of the PR. To be decided later maybe.

Reply:
don't understand :(

Reply:
I'm against this suggested change, it puts too much focus on the unimportant part of the example.